Config.mak

```diff
@@ -2,12 +2,11 @@
 # (only with dmd-transitional, no effect on other compilers)
 export ASSERT_ON_STOMPING_PREVENTION=1
 
-# Common D compiler flags
-override DFLAGS += -w -version=GLIBC
-
-# Enable coverage report in CI
-ifdef CI
-COVFLAG := -cov
-endif
+# Makd auto-detects if Ocean's test runner should be used based on submodules,
+# but we -or course- don't have Ocean as a submodule, so we set it explicitly.
+TEST_RUNNER_MODULE := ocean.core.UnitTestRunner
 
+# By default MakD looks for "integrationtest" folder
 INTEGRATIONTEST := test
+
+
```

Comment:
Trailing whitespace :o

Also, maybe the comment on that module should be a copy instead of a link to a private repo